lib/ReplaceSource.js

```diff
@@ -157,11 +154,7 @@ class ReplaceSource extends Source {
 			line: 1,
 			column: 0
 		});
-		if(original) {
-			return new SourceNode(original.line, original.column, original.source, newString);
-		} else {
-			return newString;
-		}
+		return original ? new SourceNode(original.line, original.column, original.source, newString) : newString
```

Comment:
not sure if this is better as line is less readable in current length, maybe its worth to
```
return original ?
  new SourceNode(original.line, original.column, original.source, newString) :
  newString
```

Reply:
Wouldn't this be easier:

```js
return original
  ? new SourceNode(original.line, original.column, original.source, newString)
  : newString
```

Reply:
webpack/webpack eslint wouldnt let you too, but maybe in this repo will suffice

Reply:
Oh, really? That sucks 😞

Reply:
Should probably keep it consistent with webpack/webpack then

Reply:
Yes it would be great to have that match our webpack/webpack repo.

Reply:
@TheLarkInn we usually host eslint config in a separate repo and then within our stack just extend it like this and if needed add some overrides below in .eslintrc:
```javascript
{
  "extends": "./node_modules/shared-repository-with-configs/.eslintrc"
}
```